WebApp/webapp/src/components/Layout/index.module.css

```diff
@@ -2,11 +2,6 @@
     display: flex;
 }
 
-.navbar {
-    flex: 1 0 10vw;
-    background-color: var(--navbar-colour);
-}
```

Comment:
keeping this in prevents a white bar from appearing when resizing the application

Reply:
fixed that without this